### PR TITLE
Updated node-sass

### PR DESCRIPTION
**Pull request by sbehrends**
Fixes errors when compiling with libsass

### CODE REVIEW

**Comment by silvenon** [comment]
Could you please squash your commits?

File: app/index.js [hunk 1]
```diff
@@ -45,15 +45,23 @@ module.exports = yeoman.generators.Base.extend({
...
         name: 'Bootstrap',
         value: 'includeBootstrap',
+        checked: false
+      },{
+        name: 'Foundation',
+        value: 'includeFoundation',
+        checked: false
+      },{
+        name: 'Bourbon',
+        value: 'includeBourbon',
```
Comment (silvenon) [inline app/index.js:55]: Why are you adding options for Foundation and Bourbon?

File: app/index.js [hunk 2]
```diff
@@ -33,7 +33,7 @@ module.exports = yeoman.generators.Base.extend({
     if (!this.options['skip-welcome-message']) {
       this.log(require('yosay')());
       this.log(chalk.magenta(
-        'Out of the box I include HTML5 Boilerplate, jQuery, and a ' +
+        'Out of the box you have HTML5 Boilerplate, jQuery, and a ' +
```
Comment (silvenon) [inline app/index.js:36]: Why this copy change?

**Comment by sindresorhus** [comment]
Please keep this PR focused to only updating grunt-sass. Also make sure the tests passes.

**Comment by sbehrends** [comment]
Yup Sorry.
Actually I was extending the webapp generator with Bourbon or Foundation. But while building it found that grunt-sass was giving unexpected errors.

The PR would be just grunt-sass update to 0.17.0 that manage errors correctly.

File: app/index.js (hunk 2 above)
Comment (sbehrends) [inline app/index.js:36]: Just for fun :)
I thought it as a collaborative fork. So the "first person" does not fit. At least would be "we included". So I change it as "you will receive this with this generator" does not matter who is giving it

File: app/index.js (hunk 1 above)
Comment (sbehrends) [inline app/index.js:55]: To be more flexbile while building. I found webapp generator the most complete (with img/svg/ minification and so on. The idea is to extend it with other frameworks. (Maybe also adding partials?)

**Comment by silvenon** [comment]
Thanks for your help. You can send small PRs like that straight from GitHub, by just clicking on the edit button when viewing a file. (You can try it now if you want.)

**Pull request closed without merging**